glean-core/src/metrics/mod.rs

```diff
@@ -22,11 +22,11 @@ pub use self::uuid::UuidMetric;
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub enum Metric {
-    String(String),
     Boolean(bool),
     Counter(i32),
-    Uuid(String),
+    String(String),
     StringList(Vec<String>),
+    Uuid(String),
```

Comment:
Just for ... alphabetic reasons?

Reply:
Mh, yes, feels weird to have a random order there